build.gradle

```diff
@@ -381,6 +382,13 @@ spotbugs {
     excludeFilter = file("./spotbugs-exclude.xml")
 }
 
+spotless {
+    ratchetFrom 'code-formatting-required'
+    java {
+        googleJavaFormat().aosp()
```

Comment:
Is the coding style ok with Android Open Source Project (AOSP) style?
https://source.android.com/setup/contribute/code-style

You can also change to the Google Java Style Guide.
https://google.github.io/styleguide/javaguide.html

I chose AOSP because the style over here had 4 spaces indentations.
This way, you can also change the indentation in the Google Java Style Guide. However, it failed to change some of the files.
```groovy
spotless {
    ratchetFrom 'code-formatting-required'
    java {
        googleJavaFormat()
        indentWithTabs(2)
        indentWithSpaces(4)
    }
}
```